CHANGES/1677.misc.rst

```diff
@@ -0,0 +1 @@
+Fixed an issue #1677
```

Comment:
What does in means for end-users in the changelog?

Changes description should be understandable for readers that wants to know what actually is changed in the update.